system/logaccess/ElasticStack/ElasticStackLogAccess.hpp

```diff
@@ -35,9 +35,32 @@
 
 #include <cpr/response.h>
 #include <elasticlient/client.h>
+#include <elasticlient/scroll.h>
 
 using namespace elasticlient;
 
+class ElasticStackLogAccess;
+
+class ELASTICSTACKLOGACCESS_API ElasticStackLogStream : public CInterfaceOf<IRemoteLogAccessStream>
+{
+private:
+    elasticlient::Scroll m_esSroller;
+
+    std::string m_queryString;
+    LogAccessLogFormat m_outputFormat;
+
+public:
+    static constexpr const char * DEFAULT_SCROLL_TIMEOUT = "1m"; //Elastic Time Units (i.e. 1m = 1 minute).
+    static constexpr std::size_t  DEFAULT_SCROLL_SIZE = 100;
+
+    virtual bool readLogEntries(StringBuffer & record) override;
```

Comment:
minor:  Normally a class would have constructors, then destuctors, then virtuals to make it easy to read.